ext/intl/grapheme/grapheme_string.c

```diff
@@ -339,11 +344,12 @@ PHP_FUNCTION(grapheme_substr)
 	int32_t (*iter_func)(UBreakIterator *);
 	bool no_length = true;
 
-	ZEND_PARSE_PARAMETERS_START(2, 3)
+	ZEND_PARSE_PARAMETERS_START(2, 4)
 		Z_PARAM_STRING(str, str_len)
 		Z_PARAM_LONG(lstart)
 		Z_PARAM_OPTIONAL
 		Z_PARAM_LONG_OR_NULL(length, no_length)
+		Z_PARAM_STRING(locale, locale_len)
```

Comment:
Please use the `PATH` ZPP specifier for all usages as the length is ignored in the underlying C++ call and a null byte inside a string will cause issues.
```suggestion
		Z_PARAM_PATH(locale, locale_len)
```

Reply:
Thanks, fixed.